content/docs/intro/concepts/_index.md

```diff
@@ -8,24 +8,22 @@ menu:
 aliases: ["/docs/reference/concepts/"]
 ---
 
-The Pulumi Cloud Development Platform is a combination of tools, libraries, runtime, and service that delivers a consistent development and operational control plane for cloud-native infrastructure.  That is, Pulumi enables you to not only manage your infrastructure as code, it enables you to define and manage your infrastructure using real programming languages and all of their supporting tools, not YAML.
+The Pulumi Cloud Development Platform is a combination of tools, libraries, runtime, and service that delivers a consistent development and operational control plane for cloud-native infrastructure.  Not only does Pulumi enable you to manage your infrastructure as code, it also lets you define and manage your infrastructure using real programming languages (and all of their supporting tools) instead of YAML.
 
-At the center of Pulumi is an open-source cloud object model, coupled with an evaluation runtime that understands how to take programs written in any language, understand the cloud resources necessary to execute them, and then plan and manage those resources in a robust way. This cloud runtime and object model is inherently language- and cloud-neutral, enabling Pulumi to support many languages and clouds rapidly.
+At the center of Pulumi is an open-source cloud object model, coupled with an evaluation runtime that knows how to take programs written in any language, understand the cloud resources necessary to execute them, and then plan and manage those resources in a robust way. This cloud runtime and object model is inherently language- and cloud-neutral, enabling Pulumi to support many languages and clouds rapidly.
 
-Each Pulumi [project]({{< relref "project.md" >}}) contains a [program]({{< relref "programming-model.md" >}}) --  executable code in a familiar language such as JavaScript, TypeScript, Python, or Go.  You will manage this program as you would any other source code, for example, using Git.
+Each Pulumi [project]({{< relref "project.md" >}}) contains a [program]({{< relref "programming-model.md" >}})---executable code in a familiar language such as JavaScript, TypeScript, Python, or Go. You will manage this program as you would any other source code. For example, by using Git.
```

Comment:
Is that triple dash how we make the em-dash?

Reply:
Yes, that's how. Two dashes make an en dash.